design-reviews/recipes/feature-story-design-review-template.md

```diff
@@ -0,0 +1,60 @@
+# Your Feature or Story Design Title Here
+
+> Does the feature re-use or extend existing patterns / interfaces that have already been established for the project?
+> Does the feature expose new patterns or interfaces that will establish a new standard for new future development?
+
+* [Feature/Story Name](http://link-to-feature-or-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview/Problem Statement
+
+* Describe the feature/story with a high-level summary.
+* Consider additional background and justification, for posterity and historical context.
+
+## Goals/In-Scope
+
+* List the goals that the feature/story will help us achieve.  
+* This should include acceptance criteria required to meet [definition of done](../../team-agreements/definition-of-done/).
+
+## Non-goals / Out-of-Scope
```

Comment:
what about declaring the assumptions that were made for this design?

Reply:
Yes that is added in addition to the background knowledge of the design.